medical-portal/src/UI/src/app/shared/api/models.ts

```diff
@@ -10,6 +10,7 @@ export { DocumentSubTypes } from './models/document-sub-types';
 export { Driver } from './models/driver';
 export { Endorsement } from './models/endorsement';
 export { Licence } from './models/licence';
+export { MedicalCondition } from './models/medical-condition';
```

Comment:
@mihailistov it looks like your MC changes snuck into Shruthi's PR

Reply:
This was added when i generated the API